File: Aulas/C/hello.c
#include <stdio.h>
#include <cs50.h> //Adicionar a biblioteca do CS50

int main(void)
{
    //get_string é uma função específica da biblioteca do CS50!
    string nome = get_string("Qual é o seu nome?\n");
    printf("Olá, %s \n" ,nome);
    
}

File: Aulas/Python/hello.py
#print("ola, mundo")
#entrada do teclado usando input:
resposta = input("Seu nome: ")
print(f"Ola, {resposta}")

from cs50 import get_string

resposta = get_string("Qual é o seu nome? ")

#print("Olá, " + resposta) outra forma de concatenar uma variavel:
print(f"Ola, {resposta}")